core-bundle/src/Resources/contao/languages/en/tl_form_field.xlf

```diff
@@ -237,7 +237,7 @@
         <source>A form field can be focused by pressing the [ALT] or [CTRL] key and the access key simultaneously.</source>
       </trans-unit>
       <trans-unit id="tl_form_field.tabindex.0">
-        <source>Tab index</source>
+        <source>Tabindex</source>
       </trans-unit>
       <trans-unit id="tl_form_field.tabindex.1">
         <source>The position of the form field in the tabbing order.</source>
```

Comment:
Should we add a note that this field is deprecated here too?

```suggestion
        <source>The position of the form field in the tabbing order. (Deprecated)</source>
```

Reply:
> That means every translator has to translate that deprecated field yet again just for it to be removed soon …

This would apply here as well then, wouldn't it?

Reply:
Yes. As I’m not involved in the translation process, I cannot evaluate cost vs. benefit here. I’m happy either way.